### PR TITLE
Persist "Toggle Sidebar" setting after quitting

### PR DESCRIPTION
**Pull request by jdanford**

### CODE REVIEW

**Comment by jdanford** [comment]
@sindresorhus any idea what's wrong with the AppVeyor build? Not sure about you, but I'm just getting this message:
> Project not found or access denied.

**Comment by sindresorhus** [comment]
Just ignore the AppVeyor failure. I've been trying to remove AppVeyor without success.

**Comment by sindresorhus** [comment]
Looks pretty good, but you need to add a default value for the config here: https://github.com/sindresorhus/caprine/blob/master/config.js Should be `true` by default.

**Comment by jdanford** [comment]
Default value has been added!

File: browser.js [hunk 1]
```diff
@@ -82,8 +77,6 @@ function setDarkMode() {
 function setVibrancy() {
 	document.documentElement.classList.toggle('vibrancy', config.get('vibrancy'));
 	ipc.send('set-vibrancy');
-
-	document.documentElement.style.backgroundColor = 'transparent';
```
Comment (sindresorhus) [inline browser.js]: I just noticed this and it breaks vibrancy mode. Please don't do unrelated changes.

Comment (sindresorhus) [reply browser.js]: I don't remember, but I think it's so it set right away instead of waiting for `vibrancy.css` to be loaded.

Comment (jdanford) [reply browser.js]: If you enabled vibrancy and then disabled it, i.e. called `setVibrancy()` twice, wouldn't the background color still be set to transparent?

Comment (sindresorhus) [reply browser.js]: I dunno. I don't have time to look into it. It's not really relevant to this regardless. Happy to have it fixed/improved in a separate PR though.

Comment (jdanford) [reply browser.js]: Alright, I removed the unrelated change, so this PR should be good to merge.

**Comment by sindresorhus** [comment]
Thanks :)
